### PR TITLE
google-apps-script CardService update

### PR DESCRIPTION
**Pull request by pierluigi-montagna**
I've added missing definitions on the CardService for the package google-apps-script.

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://developers.google.com/apps-script/reference/card-service/card-service>>

### CODE REVIEW

**Review by PopGoesTheWza — CHANGES_REQUESTED** [review]
Please use an indentation of two spaces, hence enabling manageable diff and making review possible.
TIA.

**Comment by pierluigi-montagna** [comment]
> Please use an indentation of two spaces, hence enabling manageable diff and making review possible.
> TIA.

I've used indentation with 2 spaces.
Just a little note: I've followed the recommended steps and it says that the indentation should be 4, is it ok ?

> Formatting: Use 4 spaces. Prettier is set up on this repo, so you can run npm run prettier -- --write

File: types/google-apps-script/google-apps-script.card-service.d.ts [hunk 1]
```diff
@@ -574,7 +636,10 @@ declare namespace GoogleAppsScript {
      * FILLED buttons have a background color you can set with
      * TextButton.setBackgroundColor(backgroundColor).
      */
-    enum TextButtonStyle { TEXT, FILLED }
+    enum TextButtonStyle {
```
Comment (PopGoesTheWza) [inline types/google-apps-script/google-apps-script.card-service.d.ts:639]: keep as single line please

File: types/google-apps-script/google-apps-script.card-service.d.ts [hunk 2]
```diff
@@ -496,7 +554,11 @@ declare namespace GoogleAppsScript {
     /**
      * Type of selection input.
      */
-    enum SelectionInputType { CHECK_BOX, RADIO_BUTTON, DROPDOWN }
+    enum SelectionInputType {
```
Comment (PopGoesTheWza) [inline types/google-apps-script/google-apps-script.card-service.d.ts:557]: keep as single line please

File: types/google-apps-script/google-apps-script.card-service.d.ts [hunk 3]
```diff
@@ -438,7 +493,10 @@ declare namespace GoogleAppsScript {
      * Using OnClose may cause OpenAs to be ignored; if the client platform cannot
      * support both selected values together, OnClose takes precedence.
      */
-    enum OpenAs { FULL_SIZE, OVERLAY }
+    enum OpenAs {
```
Comment (PopGoesTheWza) [inline types/google-apps-script/google-apps-script.card-service.d.ts:496]: keep as single line please

File: types/google-apps-script/google-apps-script.card-service.d.ts [hunk 4]
```diff
@@ -427,7 +479,10 @@ declare namespace GoogleAppsScript {
      * The implementation depends on the client platform capabilities. OnClose may cause OpenAs to be ignored; if the client platform cannot support both selected values together,
      * OnClose takes precedence.
      */
-    enum OnClose { NOTHING, RELOAD_ADD_ON }
+    enum OnClose {
```
Comment (PopGoesTheWza) [inline types/google-apps-script/google-apps-script.card-service.d.ts:482]: keep as single line please

File: types/google-apps-script/google-apps-script.card-service.d.ts [hunk 5]
```diff
@@ -390,7 +439,10 @@ declare namespace GoogleAppsScript {
     /**
      * An enum type that specifies the type of loading or progress indicator to display while an Action is being processed.
      */
-    enum LoadIndicator { SPINNER, NONE }
+    enum LoadIndicator {
```
Comment (PopGoesTheWza) [inline types/google-apps-script/google-apps-script.card-service.d.ts:442]: keep as single line please

File: types/google-apps-script/google-apps-script.card-service.d.ts [hunk 6]
```diff
@@ -350,7 +396,10 @@ declare namespace GoogleAppsScript {
     /**
      * An enum that defines an image cropping style.
      */
-    enum ImageStyle { SQUARE, CIRCLE }
+    enum ImageStyle {
```
Comment (PopGoesTheWza) [inline types/google-apps-script/google-apps-script.card-service.d.ts:399]: keep as single line please

File: types/google-apps-script/google-apps-script.card-service.d.ts [hunk 7]
```diff
@@ -305,15 +312,54 @@ declare namespace GoogleAppsScript {
...
+    enum ContentType {
+      TEXT,
+      MUTABLE_HTML,
+      IMMUTABLE_HTML,
+    }
     /**
      * Predefined icons that can be used in various UI objects, such as ImageButton or KeyValue widgets.
      */
-    enum Icon { NONE, AIRPLANE, BOOKMARK, BUS, CAR, CLOCK, CONFIRMATION_NUMBER_ICON, DOLLAR, DESCRIPTION, EMAIL, EVENT_PERFORMER, EVENT_SEAT, FLIGHT_ARRIVAL, FLIGHT_DEPARTURE, HOTEL, HOTEL_ROOM_TYPE, INVITE, MAP_PIN, MEMBERSHIP, MULTIPLE_PEOPLE, OFFER, PERSON, PHONE, RESTAURANT_ICON, SHOPPING_CART, STAR, STORE, TICKET, TRAIN, VIDEO_CAMERA, VIDEO_PLAY }
+    enum Icon {
```
Comment (PopGoesTheWza) [inline types/google-apps-script/google-apps-script.card-service.d.ts:330]: keep as single line please

File: types/google-apps-script/google-apps-script.card-service.d.ts [hunk 8]
```diff
@@ -305,15 +312,54 @@ declare namespace GoogleAppsScript {
...
-    enum ComposedEmailType { REPLY_AS_DRAFT, STANDALONE_DRAFT }
+    enum ComposedEmailType {
+      REPLY_AS_DRAFT,
+      STANDALONE_DRAFT,
+    }
     /**
      * An enum value that specifies the content type of the content generated by a UpdateDraftActionResponse.
      */
-    enum ContentType { TEXT, MUTABLE_HTML, IMMUTABLE_HTML }
+    enum ContentType {
```
Comment (PopGoesTheWza) [inline types/google-apps-script/google-apps-script.card-service.d.ts:322]: keep as single line please

File: types/google-apps-script/google-apps-script.card-service.d.ts [hunk 9]
```diff
@@ -305,15 +312,54 @@ declare namespace GoogleAppsScript {
     /**
      * An enum value that specifies whether the composed email is a standalone or reply draft.
      */
-    enum ComposedEmailType { REPLY_AS_DRAFT, STANDALONE_DRAFT }
+    enum ComposedEmailType {
```
Comment (PopGoesTheWza) [inline types/google-apps-script/google-apps-script.card-service.d.ts:315]: keep as single line please

File: types/google-apps-script/google-apps-script.card-service.d.ts [hunk 10]
```diff
@@ -9,6 +9,8 @@
 
 declare namespace GoogleAppsScript {
   namespace Card_Service {
+    import ConferenceData = GoogleAppsScript.Calendar.Schema.ConferenceData;
```
Comment (PopGoesTheWza) [inline types/google-apps-script/google-apps-script.card-service.d.ts:12]: avoid `import` statements in `.d.ts` files.

insert between lines 7 & 8 the following triple-slash directive:

```typescript
/// <reference path="google-apps-script.conference-data.d.ts" />
```

File: types/google-apps-script/google-apps-script.card-service.d.ts [hunk 11]
```diff
@@ -686,16 +751,79 @@ declare namespace GoogleAppsScript {
...
+      printJson(): string;
+    }
+
+    /**
+     * A builder for CalendarEventActionResponse objects.
+     */
+    interface CalendarEventActionResponseBuilder {
+      addAttendees(emails: string[]): CalendarEventActionResponseBuilder;
+      build(): CalendarEventActionResponse;
+      setConferenceData(conferenceData: ConferenceData): CalendarEventActionResponseBuilder;
```
Comment (PopGoesTheWza) [inline types/google-apps-script/google-apps-script.card-service.d.ts:775]: should be `setConferenceData(conferenceData: Conference_Data.ConferenceData): CalendarEventActionResponseBuilder;`

**Review by PopGoesTheWza — CHANGES_REQUESTED** [review]
Have you used <https://github.com/PopGoesTheWza/dts-google-apps-script> to assist you in writing this PR?

**Comment by pierluigi-montagna** [comment]
> Have you used https://github.com/PopGoesTheWza/dts-google-apps-script to assist you in writing this PR?
Sorry didn't know about the tool ! really helpful, probably missed it in the readme. 
It did a lot of changes on other files though, so I've just taken what I need and kept the style and everything else as it was before :)

**Review by PopGoesTheWza — APPROVED** [review]

**Comment by PopGoesTheWza** [comment]
@pierluigi-montagna sorry for the late review.

**Review by PopGoesTheWza — APPROVED** [review]

**Pull request merged**